components/cv_form/additional_info.tsx

```diff
@@ -2,7 +2,22 @@ import { Field, FieldArray } from "formik";
 import CvFieldArray from "./cv_field_array";
 import * as Icons from "../Icons";
 
-export function AdditionalInfo({ formProps }: any) {
+interface AdditionalInfoProps {
+  formProps: any;
+  setCertificationsToRemove: (cb: (prevState: string[]) => string[]) => void;
+}
+
+export function AdditionalInfo({
+  formProps,
+  setCertificationsToRemove,
+}: AdditionalInfoProps) {
+  const onRemove = (certificationId: string) => {
+    setCertificationsToRemove((prevIds: Array<string>) => [
```

Comment:
nit: `prevIds` should be inferred as string[] from `AdditionalInfoProps` type you've added.

```suggestion
    setCertificationsToRemove((prevIds) => [
```